project/plugins.sbt

```diff
@@ -2,7 +2,7 @@ DefaultOptions.addPluginResolvers
 resolvers += Resolver.typesafeRepo("releases")
 
 addSbtPlugin("com.typesafe.sbt" % "sbt-web" % "1.4.3")
-addSbtPlugin("org.jetbrains" % "sbt-idea-plugin" % "2.1.3")
+//addSbtPlugin("org.jetbrains" % "sbt-idea-plugin" % "2.1.3")
```

Comment:
We should either uncomment or delete this, I think, but that can happen in a follow-up.